sessionctx/variable/varsutil.go

```diff
@@ -85,6 +85,8 @@ func GetSessionOnlySysVars(s *SessionVars, key string) (string, bool, error) {
 		return string(j), true, nil
 	case TiDBForcePriority:
 		return mysql.Priority2Str[mysql.PriorityEnum(atomic.LoadInt32(&ForcePriority))], true, nil
+	case TiDBSlowLogThreshold:
+		return strconv.FormatUint(atomic.LoadUint64(&config.GetGlobalConfig().Log.SlowThreshold), 10), true, nil
```

Comment:
IMHO, `adapator.go:351` also need change to `atomic.LoadUnit64` to keep visibility guarantee?

Reply:
also better attach val to a session variable to reduce high rate atomic operation.